customer-commons/src/test/resources/features/get_all_customers.feature

```diff
@@ -0,0 +1,40 @@
+Feature: Get all Customers
+
+  Scenario: The User requests all Customers
+    Given that the User is looking for all Customers
+    When they set the Accept header to "application/json"
+    And they request GET /customer
+    Then they receive a response with status code 200
+    And they see that the response Content-Type header is "application/json"
+    And they see that the response body is a JSON object with a list of Customers
```

Comment:
"JSON array containing customer objects", and I think the other feature for customer covers the checks here